SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.h

```diff
@@ -388,7 +413,9 @@ class SiPixelDigitizerAlgorithm  {
 	       const TrackerTopology *tTopo,
                const std::vector<EnergyDepositUnit>& ionization_points,
                std::vector<SignalPoint>& collection_points) const;
-    void induce_signal(const PSimHit& hit,
+    void induce_signal(std::vector<PSimHit>::const_iterator inputBegin,
```

Comment:
Since we are at it, it may be useful to change the name of this method, since it does not actually induce any signal.  (The charges are induced in the actual charge collection, but this code does not do it.)

Reply:
Here I actually disagree. I see your point, that in a physical sensor the signal is induced during the drift. However, in most simulations (including this one) the charges are first propagated and then, depending on their final position, assigned to a pixel. In the latter step, the conversion from a collected charge cloud to a per-pixel signal is done, thus I think that `induce_signal` is an appropriate name for this method. One could improve by using `calculate_signal`, but I don't see an urgent need.